.kitchen.yml

```diff
@@ -12,29 +12,60 @@ provisioner:
   data_bags_path: test/data_bags
 
 platforms:
+  # Ubuntu 18.04 is what’s used on the live OSM servers,
+  # according to https://hardware.openstreetmap.org/
   - name: ubuntu-18.04
+  - name: ubuntu-18.04-ci
+    # Driver, transport, and provisioner using Docker and Chef Infra Client,
+    # necessary when used on virtual CI boxes that refuse to run Virtualbox or Vagrant.
+    # https://github.com/someara/kitchen-dokken
+    driver:
+      name: dokken
+      image: dokken/ubuntu-18.04
+      provision: true
+      # Docker and systemd need full capabilities granted in privileged mode
+      privileged: true
```

Comment:
I tried dokken/ubuntu-18.04 with privileged: true and it always seemed to interfere with my local system, i.e. logging off users, interfering with X server, etc. Curious if you experience something similar when running this locally?

Reply:
I'm running in an ssh session on errol and haven't noticed anything odd, but there's no X or anything there.

Reply:
I haven’t experienced anything like this when testing on Mac OS X 10.14.